src/org/zaproxy/zap/extension/scripts/ConsolePanel.java

```diff
@@ -152,6 +152,19 @@ private String getSyntaxForScript (String engine) {
 		}
 	}
 
+	private String getSyntaxForExtension (String name) {
+		String nameLc = name.toLowerCase();

```

Comment:
Should be used `.toLowerCase(Locale.ROOT);` to avoid locale issues. Worth updating `getSyntaxForScript`.